### PR TITLE
docs: Prepping for 1.0.0 release

### PR DESCRIPTION
**Pull request by brianredbeard**
As a a part of the 1.0.0 release we want to make sure that we have a
valid changelog documenting what has been added, removed, and changed in
the project.

### CODE REVIEW

**Comment by brianredbeard** [comment]
This is not ready to merge.  Starting to solicit feedback.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## [0.1.0] 2014-10-20
+### Added
+- Collection of disk statistics from host
+- Collection of memory statistics from host
+- Added network related commands
+- Adding general OS collection commands
+
+[Unreleased]: https://github.com/coreos/mayday/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/coreos/mayday/compare/455bde42...v0.1.0
+# vim:set ts=2 sw=2 expandtab:
```
Comment (brianredbeard) [inline CHANGELOG.md]: Need to choose a different character for the modeline

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) 
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+### Added
+- Data collection can now be specified through a config file
+  - Config file can be specified via the arguments `-c` & `--config`
+  - Config files specify a list of files and commands to be collected
+  - A config file can be referenced from it's default locations as a "profile"
```
Comment (brianredbeard) [inline CHANGELOG.md]: flag missing for profile

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    allowing for users to keep custom sets of telemetry for different products
+- systemd style directory preference support for config files/profiles
+  - `/etc/mayday`
+  - `/usr/share/mayday`
+- "danger" flag which will gather more logfiles and _potentially_
```
Comment (brianredbeard) [inline CHANGELOG.md]: flag missing for danger

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  sensitive information
+- Output file selection
```
Comment (brianredbeard) [inline CHANGELOG.md]: flag missing for output

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- Improved ASCII artwork
+- Collection specification moved into configuration file
+- Removed godeps
+- Golang vendoring is now done through [glide](https://github.com/Masterminds/glide)
+- Improved error messaging
+- Fille collection and tar creation now executes in memory
+- Updated yaml v2 vendored package
+- Flag mechanism now uses [viper](https://github.com/spf13/viper)
+
+
```
Comment (brianredbeard) [inline CHANGELOG.md]: Extra space

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  - rkt support
+  - docker support
+- Collection of systemd info
+- Journal collection for system supplied units
+
+### Changed
+- Improved ASCII artwork
+- Collection specification moved into configuration file
+- Removed godeps
+- Golang vendoring is now done through [glide](https://github.com/Masterminds/glide)
```
Comment (tschuy) [inline CHANGELOG.md]:
```
- Golang vendoring
+ Dependency vendoring
```
?

**Comment by tschuy** [comment]
Other than that minor wording change, 

lgtm

**Pull request merged**